build-parent/pom.xml

```diff
@@ -81,13 +81,13 @@
         <junit-pioneer.version>2.2.0</junit-pioneer.version>
 
         <!-- Database images for JDBC/Reactive/Hibernate tests and devservices -->
-        <postgres.image>docker.io/postgres:14</postgres.image>
+        <postgres.image>docker.io/postgres:16</postgres.image>
         <mariadb.image>docker.io/mariadb:10.11</mariadb.image>
         <db2.image>icr.io/db2_community/db2:11.5.9.0</db2.image>
         <mssql.image>mcr.microsoft.com/mssql/server:2022-latest</mssql.image>
-        <mysql.image>docker.io/mysql:8.0</mysql.image>
+        <mysql.image>docker.io/mysql:8.0.33</mysql.image>
```

Comment:
Afaik wouldn't be better to update mysql to 8.4 as it is newest LTS version by
https://endoflife.date/mysql and 
 https://dev.mysql.com/blog-archive/introducing-mysql-innovation-and-long-term-support-lts-versions/

Reply:
Thanks I just bumped to 8.4

Reply:
Fine by me but you're taking risks, ORM 6.5 is getting tested against 8.2: https://github.com/hibernate/hibernate-orm/blob/5cd7eb2260db2e8e9c1dd7c5b600ba8a5ae95c77/docker_db.sh#L18-L20

I'll suggest they bump that upstream though.